internal/service/project/data_source_project_test.go

```diff
@@ -24,13 +24,13 @@ func TestAccProjectDSProject_byID(t *testing.T) {
 		Steps: []resource.TestStep{
 			{
 				Config: testAccMongoDBAtlasProjectDSByIDUsingRS(acc.ConfigProject(projectName, orgID,
-					[]*matlas.ProjectTeam{
+					[]*admin.TeamRole{
 						{
-							TeamID:    acc.GetProjectTeamsIdsWithPos(0),
+							TeamId:    conversion.StringPtr(acc.GetProjectTeamsIdsWithPos(0)),
```

Comment:
is there always an element in position 0?

Reply:
Yes, this change is only for the type from old SDK to new SDK, but values & what is being tested remains unchanged.